cmd/retention-policy-agent/main.go

```diff
@@ -0,0 +1,107 @@
+/*
+Copyright 2024 The Tekton Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package main
+
+import (
+	"database/sql"
+	"fmt"
+	"time"
+
+	"go.uber.org/zap"
+
+	"github.com/tektoncd/results/pkg/api/server/config"
+	"github.com/tektoncd/results/pkg/api/server/logger"
+	"github.com/tektoncd/results/pkg/retention"
+	_ "go.uber.org/automaxprocs"
+	"gorm.io/driver/postgres"
+	"gorm.io/gorm"
+
+	_ "k8s.io/client-go/plugin/pkg/client/auth"
+	_ "knative.dev/pkg/client/injection/kube/client"
+
+	gormlogger "gorm.io/gorm/logger"
+	"k8s.io/apimachinery/pkg/util/wait"
```

Comment:
nit: these import could be re-organized 🙃

Reply:
Re-Organised them.